api/resources_portal/migrations/0001_initial.py

```diff
@@ -734,6 +720,25 @@ class Migration(migrations.Migration):
                 to="resources_portal.Material",
             ),
         ),
+        migrations.AddField(
+            model_name="user",
+            name="grants",
+            field=models.ManyToManyField(
+                through="resources_portal.GrantUserAssociation", to="resources_portal.Grant"
+            ),
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="groups",
+            field=models.ManyToManyField(
+                blank=True,
+                help_text="The groups this user belongs to. A user will get all permissions granted to each of their groups.",
+                related_name="user_set",
+                related_query_name="user",
+                to="auth.Group",
+                verbose_name="groups",
+            ),
+        ),
```

Comment:
Same comment as the other PR, I think we should switch to adding rather than replacing migrations. Also note that you've got several PRs making migrations simultaneously so you'll need to merge and regenerate them one by one. (I know it's at least two, it might even be more though.)